doc/api/next_api_changes/removals/24254-OG.rst

```diff
@@ -0,0 +1,64 @@
+Removal of deprecations in the Selector widget API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RectangleSelector and EllipseSelector
+.....................................
+
+The *drawtype* keyword argument to `~matplotlib.widgets.RectangleSelector` is
+removed. From now on, the only behaviour will be ``drawtype='box'``.
+
+Support for ``drawtype=line`` is removed altogether. As a
+result, the *lineprops* keyword argument to
+`~matplotlib.widgets.RectangleSelector` is also removed.
+
+To retain the behaviour of ``drawtype='none'``, use ``rectprops={'visible':
+False}`` to make the drawn `~matplotlib.patches.Rectangle` invisible.
+
+Cleaned up attributes and arguments are:
+
+- The ``active_handle`` attribute has been privatized and removed.
+- The ``drawtype`` attribute has been privatized and removed.
+- The ``eventpress`` attribute has been privatized and removed.
+- The ``eventrelease`` attribute has been privatized and removed.
+- The ``interactive`` attribute has been privatized and removed.
+- The *marker_props* argument is removed, use *handle_props* instead.
+- The *maxdist* argument is removed, use *grab_range* instead.
+- The *rectprops* argument is removed, use *props* instead.
+- The ``rectprops`` attribute has been privatized and removed.
+- The ``state`` attribute has been privatized and removed.
+- The ``to_draw`` attribute has been privatized and removed.
+
+PolygonSelector
+...............
+
+- The *line* attribute is removed. If you want to change the selector artist
+  properties, use the ``set_props`` or ``set_handle_props`` methods.
+- The *lineprops* argument is removed, use *props* instead.
+- The *markerprops* argument is removed, use *handle_props* instead.
+- The *maxdist* argument and attribute is removed, use *grab_range* instead.
+- The *vertex_select_radius* argument and attribute is removed, use
+  *grab_range* instead.
+
+SpanSelector
+............
+
+- The ``active_handle`` attribute has been privatized and removed.
+- The ``eventpress`` attribute has been privatized and removed.
+- The ``eventrelease`` attribute has been privatized and removed.
+- The *maxdist* argument and attribute is removed, use *grab_range* instead.
+- The ``pressv`` attribute has been privatized and removed.
+- The ``prev`` attribute has been privatized and removed.
+- The ``rect`` attribute has been privatized and removed.
+- The *rectprops* argument is removed, use *props* instead.
```

Comment:
```suggestion
- The *rectprops* parameter has been renamed to *props*.
```
Note that there is a distinction "A parameter is the variable which is part of the method's signature (method declaration). An argument is an expression used when calling the method.", although that is not always used consistently. But parameter is more correct here, and it's also what we call the respective section in the docs.

Reply:
Again, C&P.